frontend/pulse3d/src/serviceWorker.js

```diff
@@ -14,6 +14,9 @@ let ClientSource = null;
 let reloadNeeded = false;
 
 const cacheName = "swCache";
+const dbName = "swDB";
```

Comment:
What's the benefit of storing the tokens this way and not in the cache?

Reply:
see slack thread